ksp/src/main/kotlin/com/tschuchort/compiletesting/Ksp.kt

```diff
@@ -117,6 +117,12 @@ var KotlinCompilation.kspLoggingLevels: Set<CompilerMessageSeverity>
     tool.loggingLevels = value
   }
 
+@ExperimentalCompilerApi
+val JvmCompilationResult.sourcesGeneratedBySymbolProcessor: Sequence<File>
+  get() = outputDirectory.parentFile.resolve("ksp/sources")
```

Comment:
I'm not thrilled about this solution but other solutions require exposing the compilation which doesn't seem ideal.

Reply:
Does the round matter here? KSP does output files by round in some circumstances

Reply:
Once you have a result all rounds should be over, so this should include all files. But perhaps I misunderstand your question?